src/components/CourseCard/CourseCard.js

```diff
@@ -1,35 +0,0 @@
-import React from 'react';
```

Comment:
Can you make sure that Storybooks and tests are deleted for deleted files?

Reply:
👍  let me know if I missed any, but I'm pretty sure I got them all!